llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -1882,7 +1878,7 @@ static bool areStridedAccessesIndependent(uint64_t Distance, uint64_t Stride,
   // Two accesses in memory (scaled distance is 4, stride is 3):
   //     | A[0] |      |      | A[3] |      |      | A[6] |      |      |
   //     |      |      |      |      | A[4] |      |      | A[7] |      |
-  return ScaledDist % Stride;
+  return Distance % Stride;
```

Comment:
It looks like you've changed the definition of `Stride` to be in bytes, not multiples of the type size. Can you add something to the comment to reflect this?